skymp5-server/cpp/lib_espm/Combiner.h

```diff
@@ -102,17 +109,23 @@ class Combiner
 
 // Helpers/utilities
 
-inline const RecordHeader* FindInGroup(const BrowserInfo& brInf,
-                                       const GroupHeader* gr,
+inline const RecordHeader* FindInGroup(const BrowserInfo& browserInfo,
+                                       const GroupHeader* group,
                                        uint32_t globFormId)
 {
+  // TODO: ideally, we should be able to get browser reference here.
```

Comment:
I think we should not leave TODOs in the codebase. They are rarely being fixed.

Reply:
I think it's simpler to just erase this function for now and re-implement it better when it's actually needed